App/Components/Interaction/CompAddFriend.razor

```diff
@@ -0,0 +1,139 @@
+@inherits UserViewComponentBase
+@implements IInteractionSubscriber
+
+@using Bamboozlers.Classes.AppDbContext
+@using Bamboozlers.Classes.Data
+@using Bamboozlers.Classes.Services
+@using Bamboozlers.Classes.Services.UserServices
+@using Bamboozlers.Classes.Utility.Observer
+@using Bamboozlers.Components.Utility
+@using Blazorise.Extensions
+
+<ModalContent Centered Size="ModalSize.Large">
+    <ModalHeader>
+        <ModalTitle>Add Friends</ModalTitle>
+    </ModalHeader>
+    <ModalBody>
+        <Heading>Find users by their username or display name.</Heading>
+        <Divider/>
+        <Div Style="display: block">
+            <div style="display: block;">
+                <TextEdit @bind-Text="SearchQuery"
+                          Style="display: inline-flex; width: 75%"
+                          ElementId="search-bar"
+                          MaskType="MaskType.RegEx"
+                          EditMask="^(?:_?[a-zA-Z0-9]+)*_?$"
+                          Placeholder="Find users by their username or display name"/>
+                <div style="display: inline-flex">
+                    <CompActionButton ActionCallback="@(async () => await PerformSearch())"
+                                      IconName="@IconName.Search"
+                                      ColumnWrapped="@false"/>
+                </div>
+            </div>
+            <div style="display: block; margin-left: 5px; margin-right: 5px;" class="settings-container scrollbar-container">
```

Comment:
`settings-container` has `padding 0 10%` which looks out of place, full-width looks better consider modals already have some padding.

Also to not, `display: block` is default for divs and doesn't need to be specified